src/app/my-info/(account)/orders/_components/Orders/OrderItemList/OrderItem/OrderItem.tsx

```diff
@@ -21,7 +21,7 @@ export default function OrderItem({ orderItem, confirmationDate, orderStatus }:
   return (
     <div className={cn('order-item')}>
       <div className={cn('item')}>
-        <ItemOverview item={orderItem} />
+        <ItemOverview item={orderItem} isroutingDetailPage />
```

Comment:
```suggestion
        <ItemOverview item={orderItem} isRoutingDetailPage />
```
사소하지만...? 비슷한 맥락으로 isDisplayOnDetailPage 도 있습니다..!